### PR TITLE
Fix for #12 non-escaped slashes in RegExp for nodejs 0.10

### PR DESCRIPTION
**Pull request by kapouer**

### CODE REVIEW

**Comment by kapouer** [comment]
This fixes #12

File: tosource.js [hunk 1]
```diff
@@ -48,3 +48,18 @@ var KEYWORD_REGEXP = /^(abstract|boolean|break|byte|case|catch|char|class|const|
 function legalKey(string) {
     return /^[a-z_$][0-9a-z_$]*$/gi.test(string) && !KEYWORD_REGEXP.test(string)
 }
+
+var isRegExpEscaped = (new RegExp('/')).toString().slice(1, -1) == "\\/";
```
Comment (marcello3d) [inline tosource.js]: I'd add a comment on why you're doing this.

**Comment by marcello3d** [comment]
Thank you for submitting a PR, definitely appreciated.

Some general codestyle issues:
* no semicolons
* use `===` instead of `==`
* and always use `{` and `}` on control blocks

Aside from that, the polyfill feels more complicated than necessary, did you explore using `RegExp.prototype.source` and `.flags`? Wondering if we need more tests for the edge cases you're covering.

**Comment by kapouer** [comment]
* `.flags` does not exist in node 0.10, but yes i made it somewhat safer.
* i don't see the point in using === for string comparison to a constant, otherwise i used it for the integer comparison

**Comment by kapouer** [comment]
Right commit is https://github.com/kapouer/node-tosource/commit/1cfdc82024ea80e040913e3bb5ee2513b02d5e2d

**Comment by marcello3d** [comment]
Looks good, thanks!
